frontend/src/components/common/AlertNotification.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, useTheme } from '@material-ui/core';
+import React from 'react';
+import { useRouteMatch } from 'react-router-dom';
+import { testAuth } from '../../lib/k8s/apiProxy';
+import { getRoutePath, ROUTES } from '../../lib/router';
+import { useTypedSelector } from '../../redux/reducers/reducers';
+
+const NOT_FOUND_ERROR_MESSAGE = 'Error: Api request error: Bad Gateway';
+
+// in ms
+const NETWORK_STATUS_CHECK_TIME = 5000;
+
+export default function AlertNotification(){
+  const routes = useTypedSelector(state => state.ui.routes);
+  const [networkStatusCheckTimeFactor, setNetworkStatusCheckTimeFactor] = React.useState(0);
+  const [error, setError] = React.useState<null | string | boolean>(null);
+  const [intervalID, setIntervalID] = React.useState<NodeJS.Timeout | null>(null);
+
+  function registerSetInterval(): NodeJS.Timeout {
+    return setInterval(() => {
+      setError(null);
```

Comment:
I think if you set the error as null here, it will force 2 refreshes: one where the error bar is removed, and another when the result from the testAuth arrives. Can you double check this?

Reply:
setting the null there means please dont show the alert notification until i have achieved the result from the testAuth method otherwise while its going to make an ajax call it will show that there is an issue we know there is an issue only after the request is made